src/components/Tool/Appeal/InitialPage/AddAppealForm/AddAppealForm.tsx

```diff
@@ -177,7 +177,7 @@ const appealFormSchema = yup.object({
     .test(
       i18n.t('Is positive?'),
       i18n.t('Must use a positive number for Initial Goal'),
-      (value) => parseFloat(value as unknown as string) >= 0,
+      (value) => typeof value === 'number' && value >= 0,
```

Comment:
When testing on https://8192-validate-percentage.d3dytjb8adxkk5.amplifyapp.com/
I was still able to enter a number with a decimal point.

Reply:
Are you saying the admin percentage validation is good but there isn't validation yet on the initial goal?

Reply:
yes that is correct